### PR TITLE
Resolved Output path of current Prisma

### PR DESCRIPTION
**Pull request by MAYANK-TRIPATH**
/claim #7771
/fixes #7771

### CODE REVIEW

File: reproductions/basic-sqlite/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Prisma development playground",
   "main": "index.ts",
   "scripts": {
+    "prisma": "node -e 'console.log(require.resolve(\"@prisma/cli\"))'",
```
Comment (rishi-raj-jain) [inline reproductions/basic-sqlite/package.json:7]: This seems like a hacky fix, why change the import path?

File: reproductions/basic-sqlite/package.json [hunk 2]
```diff
@@ -17,7 +18,7 @@
   },
   "devDependencies": {
     "@types/node": "17.0.45",
-    "prisma": "../../packages/cli",
+    "prisma": "^5.2.0",
```
Comment (aqrln) [inline reproductions/basic-sqlite/package.json:21]: Please don't change this, it must use the local package.

File: reproductions/basic-sqlite/package.json (hunk 1 above)
Comment (aqrln) [inline reproductions/basic-sqlite/package.json:7]: This package is not part of Prisma itself, it's merely a template for a local playground used for development. Changing anything here has no effect for users.

File: reproductions/basic-sqlite/package.json (hunk 2 above)
Comment (jharrell) [inline reproductions/basic-sqlite/package.json:21]: We would like to keep the relative import as that tests against the most up to date code.

File: reproductions/basic-sqlite/package.json (hunk 1 above)
Comment (jharrell) [reply reproductions/basic-sqlite/package.json:7]: I agree.

**Review by jharrell — CHANGES_REQUESTED** [review]

File: reproductions/basic-sqlite/yarn.lock [hunk 3]
```diff
@@ -0,0 +1,139 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (aqrln) [inline reproductions/basic-sqlite/yarn.lock:1]: Stray `yarn.lock` file in a pnpm workspace. If you need to add a test that specifically uses yarn for any reason, you could add support for yarn packages in `packages/client/tests/e2e` and create a test there, but I don't believe that should be necessary. Reading the issue, a regular test in `packages/cli/src/__tests__/commands/Version.test.ts` should be sufficient.

**Review by aqrln — CHANGES_REQUESTED** [review]
Thank you for opening the PR and trying to tackle the issue, however, as it currently stands, it doesn't actually change anything in Prisma and only touches unrelated files. Please let us know if anything needs further clarifications in the issue so we can help you succeed 🚀

**Comment by MAYANK-TRIPATH** [comment]
@aqrln @jharrell @rishi-raj-jain Thank you so much you reviewing this PR and guiding me.
Sorry for misunderstanding i guess it was just that we are encountering issues with inconsistent Prisma versions when running `yarn prisma -v` and need to ensure consistent Prisma versions across different environments. 
So i though its just some scripts and dependency faults.
Closing this PR for now and will raise new one with right solution :)
Thank you.

**Pull request closed without merging**